qcodes_contrib_drivers/drivers/Standa/Standa_10MWA168.py

```diff
@@ -0,0 +1,206 @@
+        self.revolution = 200.
+        self.distance = self.revolution / 8.
+
+        # initialization
+        self.serial_number = serial_number
+        device_enumeration = self.libximc.enumerate_devices(0)
+        enumeration_name = self.libximc.get_device_name(device_enumeration, 0)
+        self.device_id = self.libximc.open_device(enumeration_name)
+
+        # add parameters
+        self.add_parameter('transmittance',
+                           set_cmd=self.set_transmittance,
+                           label='Transmittance',
+                           val_mapping={
+                               1: 0, 0: 1, 9.0e-1: 2, 8.0e-1: 3, 7.2e-1: 4, 4.0e-1: 5, 3.0e-1: 6, 2.7e-1: 7,
+                               1.5e-1: 8, 1.0e-1: 9, 8.0e-2: 10, 3.0e-2: 11, 2.7e-2: 12, 1.5e-2: 13, 1.0e-2: 14,
+                               3.0e-3: 15, 2.7e-3: 16, 1.5e-3: 17, 1.0e-3: 18, 8.0e-4: 19, 3.0e-4: 20, 2.7e-4: 21,
+                               1.5e-4: 22, 1.0e-4: 23, 8.0e-5: 24, 3.0e-5: 25, 1.5e-5: 26, 3.0e-6: 27, 3.0e-7: 28,
+                               3.0e-8: 29, 3.0e-9: 30})
+
+        self.add_parameter('position',
+                           set_cmd=self.set_position,
+                           get_cmd=self.get_position,
+                           get_parser=float,
+                           label='Position')
+
+        self.add_parameter('status',
+                           get_cmd=self.get_status,
+                           get_parser=int,
+                           label='status')
+
+        self.connect_message()
+
+    # get methods
+    def get_position(self):
+        position = GetPosition()
+        self.libximc.get_position(self.device_id, ctypes.byref(position))
+        return position.Position
+
+    def get_status(self):
+        status = Status()
+        self.libximc.get_status(self.device_id, ctypes.byref(status))
+        return status.MoveSts
+
+    # set methods
+    def set_position(self, position):
+        self.libximc.command_move(self.device_id, int(position), 0)
+
+    def set_transmittance(self, transmittance_id):
+
+        # get filter to set
+        filter_wheel_1 = self.filter_wheel_1[transmittance_id]
+        filter_wheel_2 = self.filter_wheel_2[transmittance_id]
+
+        # get current position
+        current_position = self.position.get()
+
+        # determine new positions
+        position_wheel_2 = self.offset_wheel_2 + self.distance * filter_wheel_2 + \
+                           np.ceil(current_position / self.revolution + 2) * self.revolution
+        position_wheel_1 = self.offset_wheel_1 + self.distance * filter_wheel_1 + \
+                           np.ceil(current_position / self.revolution + 2) * self.revolution
+
+        if position_wheel_1 > position_wheel_2:
+            print('new', position_wheel_1, position_wheel_2)
+            position_wheel_1 -= self.revolution
+
+        # set position of the second wheel
+        self.position.set(int(position_wheel_2))
```

Comment:
perhaps `np.floor` or python's `floor` functions were meant to be used instead of `int`? those express the purpose in a much cleaner way (if the purpose is to round down)